examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -286,11 +398,11 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
       $scope.controls.custom.push(info);
 
       loadGeoJsonFiles(onEachFeature);
-      if ($scope.status.logicLevel === "city"){
-        loadCityJsonByBound(onEachFeature);
-      }
 
-      $scope.$on("leafletDirectiveMap.zoomend", function() {
+      if ($scope.zoomfunction) {
+        $scope.zoomfunction()
+      }
+      $scope.zoomfunction = $scope.$on("leafletDirectiveMap.zoomend", function() {
```

Comment:
Line 401:403 - what does this do?

Reply:
It removes the original function associate with zoom event. If we don't do this, there will be multiple functions associate with zoom or drag event, so one single zoom/drag can trigger multiple queries.

Reply:
Then, can you add this explanation as a comment in the code?

Reply:
Okay, done.